proto/v1/plan_service.proto

```diff
@@ -88,6 +88,13 @@ service PlanService {
     option (bytebase.v1.permission) = "bb.planCheckRuns.run";
     option (bytebase.v1.auth_method) = IAM;
   }
+
+  rpc PreviewPlan(PreviewPlanRequest) returns (PreviewPlanResponse) {
+    option (google.api.http) = {
+      post: "/v1/{project=projects/*}:previewPlan"
```

Comment:
How about `/v1/{project=projects/*}/plan:preview`

Reply:
Vote for `/v1/{parent=projects/*}/plans:preview`